scripts/Dockerfile.devel

```diff
@@ -1,12 +1,19 @@
 FROM ceph/ceph:v15
 
+RUN \
```

Comment:
is `\` required here?

Reply:
yeah.. requried @Madhu-1

Reply:
This is not required, can you please remove it

Reply:
Done.